Loop/Managers/DeviceDataManager.swift

```diff
@@ -364,6 +370,38 @@ final class DeviceDataManager: CarbStoreDelegate, CarbStoreSyncDelegate, DoseSto
         }
     }
 
+    /// NonMySentry Battery Calculation for Alkaline and Lithuim  #141
+    ///
+    /// - parameter currVoltage: Current Voltage Reading from Pump
+    ///
+    public var x22BatteryPercentRemaining : Double = -1
+    
+    private func setBatteryStatusforNonMySentryPumps(currVoltage : Double){
+        var minVoltage : Double
+        var maxVoltage : Double
+        var batteryNotification : Double
+     
+        // if Lithium set min and max linear voltages
+        if (self.batteryChemistry == .lithium){
+            minVoltage = 1.32
+            maxVoltage = 1.58
+            batteryNotification = 0.12
```

Comment:
The intent of this Notification was to "warn" a user of an impending battery failure or intermittent communications failures due to low battery.  This is a 8 to 12 hour warning prior to battery = 0.

https://docs.google.com/spreadsheets/d/19FpO5YO5ewQrdph_4iIXuear6xRGzl-AYTiFeHwPcVk/edit#gid=246762532.